components/FinancialSummary/SuccessStories.tsx

```diff
@@ -0,0 +1,48 @@
+import TextLink from '../typography/TextLink';
+
+const SuccessStories: React.FC = () => {
+  return (
+    <div className='bg-purple-100 px-4 sm:px-6 lg:px-8'>
+      <div className='my-16 grid lg:grid-cols-9 lg:gap-8 lg:text-center'>
+        <div className='col-span-7 col-start-2 my-12'>
+          <div className='mx-2'>
+            <h1 id='success-stories' className='m-3 text-center text-4xl font-bold'>
+                            Success Stories
+            </h1>
+            <p className='mx-auto my-3 max-w-4xl text-center text-lg text-darkGunMetal'>
+                            Thanks to financial support we can already see many<br className='hidden lg:inline-block' /> success stories in
+                            the project.
+            </p>
+          </div>
+          <div className='mt-4 grid grid-cols-1 gap-4 md:grid-cols-2 lg:grid-cols-3'>
+            <div className='m-4 p-2'>
+              <h1 className='mb-2 text-2xl font-semibold'>Community Manager</h1>
+              <p className='text-base text-darkGunMetal'>
+                                With the addition of a dedicated Community Manager, we now have a monthly newsletter,
+                                regular status updates, an active social media presence, and the ability to drive
+                                initiatives such as event organization. Dedicated focus enables us to also focus on  <TextLink href='https://github.com/orgs/asyncapi/discussions/948' target='_blank' className='text-violet'>a year to year vision</TextLink>.
+              </p>
+            </div>
+            <div className='m-4 p-2'>
+              <h1 className='mb-2 text-2xl font-semibold'>AsyncAPI Mentorship</h1>
+              <p className='text-base text-darkGunMetal'>
+                                The 2022 mentorship program yielded significant achievements: Kafka support in
+                                Glee, a centralized platform for sharing AsyncAPI tools, and a versatile error
+                                handling library for multiple projects.
+              </p>
+            </div>
+            <div className='m-4 p-2'>
+              <h1 className='mb-2 text-2xl font-semibold'>AsyncAPI Conference</h1>
+              <p className='text-base text-darkGunMetal'>
+                                Every year we organize a conference that attracts many participants. In 2022
+                                the online conference generated <TextLink href='https://www.youtube.com/playlist?list=PLbi1gRlP7pijRiA32SU36hD_FW-2qyPhl' target='_blank' className='text-violet'>3k views</TextLink>. In 2023 we organized <TextLink href='https://conference.asyncapi.com' target='_blank' className='text-violet'>four different in person events</TextLink>, some that was also live streamed.
+              </p>
+            </div>
```

Comment:
Don't make repetitive code inside the files. Use another data file to store the hardcoded data.